ec2/spark_ec2.py

```diff
@@ -815,13 +804,11 @@ def deploy_files(conn, root_dir, opts, master_nodes, slave_nodes, modules):
     cluster_url = "%s:7077" % active_master
 
     if "." in opts.spark_version:
-        # Pre-built spark & shark deploy
-        (spark_v, shark_v) = get_spark_shark_version(opts)
+        # Pre-built Spark deploy
+        spark_v = opts.spark_version.replace("v", "")
```

Comment:
@shivaram @JoshRosen Do you know why we allow versions to be passed in as `v1.2.0` (with a leading `v`)? Is it safe to remove this option?